SU2_CFD/src/fluid/CCoolProp.cpp

```diff
@@ -56,28 +56,51 @@ void CCoolProp::SetTDState_rhoe(su2double rho, su2double e) {
   dTdrho_e = fluid_entity->first_partial_deriv(CoolProp::iT, CoolProp::iDmass, CoolProp::iUmass);
   dTde_rho = fluid_entity->first_partial_deriv(CoolProp::iT, CoolProp::iUmass, CoolProp::iDmass);
   if (fluid_entity->phase() == 6) {
```

Comment:
Is there a constant like CoolProp::iphase_gas that you can use instead of "6"?
```suggestion
  if (fluid_entity->phase() == CoolProp::iphase_?) {
```

Reply:
Thanks for you advice.

I change "if (fluid_entity->phase() == 6)" to "if (fluid_entity->phase() == CoolProp::iphase_twophase)".

Now the code should be more meaningful.